conanfile.txt

```diff
@@ -1,4 +1,6 @@
 [requires]
+seats-service-sdk/auto
```

Comment:
@doosuu: Should `auto` be replaced?

Reply:
Any suggestions? `generated` might also be an option